package.json

```diff
@@ -60,7 +60,9 @@
     "sass-loader": "^6.0.6",
     "webpack": "^3.5.6",
     "webpack-bundle-tracker": "0.2.0",
-    "webpack-dev-server": "^2.7.1"
+    "webpack-dev-middleware": "^1.12.0",
+    "webpack-dev-server": "^2.9.4",
+    "webpack-hot-middleware": "^2.20.0"
```

Comment:
Do these lines above belong to this pull request or are they related to your local development process?

Reply:
Missed that, but shouldn't this be in .gitignore then?

Reply:
`package.json` is part of the project. Package managers often add new entries automatically when installing a new package. When adding a package on purpose, it should be added as well in `package.json`, otherwise, it should remain unchanged.

Reply:
ack